cas-server-webapp/src/main/webapp/WEB-INF/casComponentsMapping.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:util="http://www.springframework.org/schema/util"
```

Comment:
I would feel better if you moved the monitor list out and back into the deployer. That is something a deployer would need to set more often the aliases, and is not so much about mapping aliases.